src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -206,46 +205,84 @@ def injectFiles(self):
                 self.sendAlert(7, msg = msg)
                 continue
 
-            myThread.transaction.begin()
-            xmlData = self.createInjectionSpec(uninjectedFiles[siteName])
-            try:
-                injectRes = self.phedex.injectBlocks(location, xmlData)
-            except HTTPException as ex:
-                # If we get an HTTPException of certain types, raise it as an error
-                if ex.status == 400:
-                    # assume it is duplicate injection error. but if that is not the case
-                    # needs to be investigated
-                    self.blocksToRecover = self.createRecoveryFileFormat(uninjectedFiles[siteName])
-                
-                msg = "PhEDEx injection failed with %s error: %s" % (ex.status, ex.result)
-                raise PhEDExInjectorPassableError(msg)
-            except Exception as ex:
-                # If we get an error here, assume that it's temporary (it usually is)
-                # log it, and ignore it in the algorithm() loop
-                msg =  "Encountered error while attempting to inject blocks to PhEDEx.\n"
-                msg += str(ex)
-                logging.error(msg)
-                logging.debug("Traceback: %s" % str(traceback.format_exc()))
-                
-                raise PhEDExInjectorPassableError(msg)
-            logging.info("Injection result: %s" % injectRes)
+            maxDatatset = 20
```

Comment:
It's not a typo if you get it wrong everywhere :-). But yes, wasn't supposed to look like this, will fix.